frontend/cypress/integration/basic-spec.ts

```diff
@@ -0,0 +1,84 @@
+import { BigNumber } from 'ethers';
+import { expect } from 'chai';
+
+describe('Collateral auctions', function () {
+    // Set the private key to one of the deterministic hardhat keys.
+    // see https://hardhat.org/hardhat-network#running-stand-alone-in-order-to-support-wallets-and-other-software
+    const KEEPER_WALLET_PRIVATE_KEY = '0x701b615bbdfb9de65240bc28bd21bbc0d996645a3dd57e7b12bc2bdf6f192c82';
+    const URL_TO_VISIT = `${process.env.FRONTEND_URL || 'localhost:3000'}/collateral?network=localhost`;
+    let daiBalanceBefore: Record<string, BigNumber>;
+    let daiBalanceAfter: Record<string, BigNumber>;
+
+    function testSwapProfit() {
+        cy.visit(URL_TO_VISIT);
+        cy.window()
+            .its('$nuxt')
+            .then(function (nuxt) {
+                nuxt.$store.dispatch('wallet/createWalletFromPrivateKey', KEEPER_WALLET_PRIVATE_KEY);
+            });
+        cy.get('table').first({ timeout: 140000 }).should('contain.text', 'ETH');
```

Comment:
Please move this number into a variable, something like `PAGE_LOAD_TIMEOUT_MS` and please use multiplication to increase readability, eg `140 * 1000` or `2.5 * 60 * 1000`

Reply:
By the way, does timeout here is the _maximum_ timeout or is it awaited anyway? With `puppeteer` the common pattern is `waitForSelector('selector', { timeout: amount_ms })` where it is awaited maximum for the `amount_ms`

Reply:
this is the maximum. as soon as the expectation is met, the flow continues.